open-craft/opencraft
# [BB-2353] Update dependencies

This is an update of Ocim's old and new frontend dependencies. 

**Testing instructions:**
1. Checkout this branch.
1. Check that Ocim's old frontend tests pass (we updated only test dependencies there).
1. Run Ocim (inside vagrant with `make run.dev`).
1. Go to `frontend/` and build the new API client with `./scripts/build-api-client.sh`
1. Run the new frontend: `npm i && npm start`.
1. Check that it doesn't break.
